data/hai/hai reveal 4 middle.txt

```diff
@@ -1498,7 +1498,56 @@ mission "Hai Reveal [A11-B] Permanent Fleets"
+			`	The election coverage is in Hai, naturally, but there's a strong sense that people are still scrambling to have relevant commentary on such short notice and with no prior opinion polling in lead-up. It certainly seems like things are heated. Different charts being presented for the same areas seem to suggest that there's a great deal of controversy and some very strong opinions being thrown around. A number of the Hai nearby make some sounds of concern occasionally, but they do not stay around to comment. Unfortunately, nothing comes up in your language by the time you've finished your food.`
+			choice
+				`	(Eat the tray.)`
+					goto tray
+				`	(Return to your ship.)`
+					goto leave
+			label tray
+			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`
+			choice
+				`	(Ask around for an update.)`
+					goto ask
+				`	(Return to your ship.)`
+					goto leave
+			label ask
+			`	When you ask around a lot of Hai don't seem to be very interested in talking to you. One in particular looks like they recognized you and actively avoided you. After you've approached a half dozen Hai unsuccessfully though, one takes pity and comes over towards you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
+			`	"Apologies for my brethren's behavior Captain, this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare that they usually would."`
+			choice
+				`	"Is politics a very important matter for the Hai?"`
+					goto politics
+				`	"What do they need to prepare for an election?"`
+			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses' yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```

Comment:
```suggestion
			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have 'their fingers on the pulse', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```

Reply:
He's meant to get it wrong, address only grammar please.

Reply:
It wasn't supposed to be that sort of change, I was even going to say that he should get it wrong, haha. 
```suggestion
			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```